app/models/donation_site.rb

```diff
@@ -22,7 +22,8 @@ class DonationSite < ApplicationRecord
 
   belongs_to :organization
 
-  validates :name, :address, presence: true
+  validates :name, :address, :phone, presence: true
```

Comment:
Phone is *not* mandatory -- this is breaking  bin/setup and a lot of the sites in production have no contact information.